### PR TITLE
Remove unused functionalities

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
Wait, shouldn't we be removing these from esqlabsR, and not from the legacy version?

**Comment by PavelBal** [comment]
Legacy should only contain what will be removed from esqlabsR

File: DESCRIPTION [hunk 1]
```diff
@@ -9,28 +9,16 @@ Authors@R:
       person("Indrajeet", "Patil", , "patilindrajeet.science@gmail.com", role = "aut",
            comment = c(ORCID = "0000-0003-1995-6531", Twitter = "@patilindrajeets")))
 Maintainer: Pavel Balazki <pavel.balazki@esqlabs.com>
-Description: Utilities functions for modeling and simulation projects
-    within 'esqLABS' organization.
+Description: Utilities package for backward compatibility of esqlbasR version >4.
```
Comment (IndrajeetPatil) [inline DESCRIPTION]:
```suggestion
Description: Utilities package for backward compatibility of 'esqlabsR' version >4.
```

**Review by IndrajeetPatil — APPROVED** [review]
LGTM!

**Pull request merged**